test/tm-cases/alerts.text

```diff
@@ -0,0 +1,15 @@
+> [!NOTE]
+> Useful information that users should know, even when skimming content.
+
+> [!TIP]
+> Helpful advice for doing things better or more easily.
+
+> [!IMPORTANT]
+> Key information users need to know to achieve their goal.
+
+> [!WARNING]
+> Urgent info that needs immediate user attention to avoid problems.
+
+> [!CAUTION]
+>
```

Comment:
Is the extra line needed here? Wasn't sure if this was to catch an edge case or not